src/pages/index.tsx

```diff
@@ -94,25 +93,24 @@ const Index: FC = () => {
     return (
         <Layout>
             <Head title="Tasks" />
-            <TasksProvider>
-                <div className={classNames.container}>
-                    {isError && <p>{TASKS_FETCH_ERROR_MESSAGE}</p>}
-                    {isLoading ? (
-                        <p>Loading...</p>
-                    ) : (
-                        <>
-                            {Object.keys(tasksGroupedByStatus).length > 0 ? (
-                                <div className={classNames.tasksContainer}>
-                                    {renderTabSection()}
-                                    {renderTaskList()}
-                                </div>
-                            ) : (
-                                !isError && <p>{NO_TASKS_FOUND_MESSAGE}</p>
-                            )}
-                        </>
-                    )}
-                </div>
-            </TasksProvider>
+
+            <div className={classNames.container}>
+                {isError && <p>{TASKS_FETCH_ERROR_MESSAGE}</p>}
```

Comment:
is it working? it returns `error`, not `isError`, check [here](https://redux-toolkit.js.org/rtk-query/overview#configure-the-store)

Reply:
yes @vinayak-trivedi it is working 
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/42600164/233683646-84554099-0ca1-43f0-ac59-db2810111ea3.png">